app/controllers/questions_controller.rb

```diff
@@ -19,9 +23,111 @@ def show
       gon.solution = solution
     end
 
+    if !(current_user.show_solutions)
+      if @room.answered(@question.id,@room.id)
+        @number_of_question = @room.question_count - @room.question_count_not_visited(@room.id) + 1
+      else
+        @number_of_question = @room.question_count - @room.question_count_not_visited(@room.id)
+      end
+    else
+      @number_of_question = @room.learning_objects.where("learning_object_id < ?",@question.id).count + 1
+    end
+
+    @feedbacks = @question.feedbacks.includes(:user)
```

Comment:
Tento riadok (`@feedbacks = @question.feedbacks.includes(:user)`), prosim, vymaz. Zrejme to zblblo pri mergovani a nasledujuci ho aj tak zneplatni.